tests/src/Unit/OgAccessHookTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Unit\OgAccessHookTest.
+ */
+
+namespace Drupal\Tests\og\Unit;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\og\OgAccess;
+
+/**
+ * @group og
+ */
+class OgAccessHookTest extends OgAccessEntityTestBase {
+
+  public function setUp() {
+    parent::setUp();
+    include_once __DIR__ . '/../../../og.module';
```

Comment:
Maybe worth a comment, maybe:

`// Since this is a unit test, we don't enable the module. However we test a hook implementation inside the module.`